setup.py

```diff
@@ -59,6 +59,8 @@
         "google-api-python-client",
         "protobuf",
         "grpcio",
+        "click",
```

Comment:
we should be able to drop click dependency as commented above

Reply:
Not sure about it - comment above